lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group_metric_filter.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: cloudwatchlogs_log_group_metric_filter
+version_added: "2.10"
+author:
+  - "Markus Bergholz (@markuman)"
+short_description: Manage CloudWatch log group metric filter
+description:
+  - Create, modify and delete Cloudwatch log group metric filter.
+  - Cloudwatch log group metric filter can be use with I(ec2_metric_alarm).
+requirements:
+  - boto3
+  - botocore
+options:
+    state:
+      description:
+        - Whether the rule is present or absent.
+      choices: ["present", "absent"]
+      required: true
+      type: str
+    log_group_name:
+      description:
+        - The name of the log group where the metric filter is applied on.
+      required: true
+      type: str
+    filter_name:
+      description:
+        - A name for the metric filter you create.
+      required: true
+      type: str
+    filter_pattern:
+      description:
+        - A filter pattern for extracting metric data out of ingested log events.
+      required: true
+      type: str
+    metric_transformation:
+      description:
+        - A collection of information that defines how metric data gets emitted.
+      type: dict
+      suboptions:
+        metric_name:
+          description:
+            - The name of the cloudwatch metric.
```

Comment:
```suggestion
            - The name of the CloudWatch metric.
```
CloudWatch being a product name we should try to get the capitalization right in our documentation.  I know I'm getting picky now, but would you mind correcting these?